src/.defaultsForValidator.js

```diff
@@ -20,6 +20,10 @@
 
 const defaults = {
   'shared': {
+    'info':{
+      'no_info': 'error',
+      'mistyped_title_or_version': 'error'
+    },
```

Comment:
Since these are explicitly required by the specification, I don't think these need to be configurable. Therefore, we don't need default values. You can remove this section